.rubocop_todo.yml

```diff
@@ -62,18 +41,11 @@ RSpec/NamedSubject:
     - 'spec/unit/config_spec.rb'
     - 'spec/unit/enforcement_spec.rb'
 
-# Offense count: 30
-# Configuration parameters: MaxNesting.
+# Offense count: 6
+# Configuration parameters: Max.
 RSpec/NestedGroups:
   Exclude:
-    - 'spec/**/*_spec.rb'
-
-# Offense count: 2
-# Configuration parameters: EnforcedStyle, SupportedStyles.
-# SupportedStyles: not_to, to_not
-RSpec/NotToNot:
-  Exclude:
-    - 'spec/unit/catalog_spec.rb'
+    - 'spec/unit/blacklight/access_controls/search_builder_spec.rb'
```

Comment:
I think we should continue to exclude everything in spec from this.  Maybe it needs to move to .rubocop.yml?